Documentation/Migration/Index.rst

```diff
@@ -20,7 +20,8 @@ and optionally also remove the old :file:`Settings.cfg` file.
 With official Docker container
 ==============================
 
-(Work in progress)
+(Work in progress - Note that we will use `:stable` at a later point to always provide stable
```

Comment:
Here too.

Reply:
I dont think we need to use a note admonition whereever we use the word "note"...

Reply:
Please check the rework.

I don't really like that it's at three places now, but that'll be gone one's the notice is gone...

Reply:
I think one note should be enough

Reply:
> I dont think we need to use a note admonition whereever we use the word "note"...

Yup, but this should be visible more prominent.